daemons/integration-test/app.py

```diff
@@ -0,0 +1,35 @@
+import os
+
+import domovoi
+import requests
+
+
+app = domovoi.Domovoi()
+
+
+@app.scheduled_function("rate(60 minutes)")
+def integration_test(*_):
```

Comment:
nitpick: prefer a more proper `method(*args, **kwargs)`

Reply:
How about `method(*_, **_)`? :)

Reply:
I switched it to `*args, **kwargs` for now though.